src/platform/Makefile.am

```diff
@@ -144,6 +144,9 @@ libDeviceLayer_a_CPPFLAGS               = \
     $(SOCKETS_CPPFLAGS)                   \
     $(NULL)
 
+nodist_libDeviceLayer_a_SOURCES         = \
```

Comment:
what use of nodist_libDeviceLayer_a_SOURCES? it seems we don't need this variable

Reply:
yep, just deleted.